indolaw-nextjs/utils/theme.tsx

```diff
@@ -1,12 +1,23 @@
 export const colors = {
-  background: "#0e7aaa",
+  trayBackground: "#0e7aaa",
+  background: "#ffffff",
   text: "#514d48",
   dark: {
     text: "#f2f2f2",
     textSecondary: "#cccccc",
   },
 };
 
+export const darkColors = {
+  trayBackground: "#1D2D50",
+  background: "#133B5C",
+  text: "#fcdab7",
+  dark: {
+    text: "#f3f3f3",
+    textSecondary: "#cccccc",
+  },
```

Comment:
@willemchua nitpick: can we rename this from 'dark' to 'tray'? (same as in colors)

since we have colors & darkColors now, having one of the fields of this object also be called 'dark' might be confusing haha